css/blogpost.scss

```diff
@@ -0,0 +1,84 @@
+@import 'styleguide';
+
+div {
+  font-family: $font-body;
+}
+
+p {
+  line-height: 2em;
+  font-size: 1em;
+  color: $offblack;
+}
+
+.title {
+  font-family: $font-heading;
+  font-weight: 800;
+  font-size: 3em;
+  color: $offblack
+}
+
+.subheading {
+  font-family: $font-heading;
+  font-weight: 600;
+  font-size: 2em;
+  margin-top: 1em;
+  margin-bottom: -.5em;
+  color: $darkgrey;
+}
+
+.date {
+  color: $grey;
+  font-size: 1.5em;
+  margin: .5em 0em;
+  font-family: $font-accent;
+}
+
+.post {
+  margin: 1em auto;
+  line-height: 1.5em;
+}
+
+.center {
+  margin: 0em auto;;
+}
+
+#container-top {
+  margin-top: 3em;
+}
+
+#container-bot {
+  margin-bottom: 6em;
+}
+
+.caption {
+  font-family: $font-accent;
+  text-align: center;
+  font-size: 1em;
+  color: #666;
+  width: 100%;
+  margin-top: 1em;
+  margin-bottom: 2em;
+}
+
+.image {
+  display: inline;
+  text-align: center;
+}
+
+img {
+  margin-top: 1.5em;
+  display: inline-block;
+}
+
+#img1 {
+  width: calc(100% * 700 / (455 + 700) - 0.1875em);
```

Comment:
I literally Googled everything and couldn't find an easy way to make multiple images of different dimensions in a row all have the same height and span the width of a div. :(